### PR TITLE
fixed situation when image was bigger then text and page just broke, and fixed paddings on mobile

### PR DESCRIPTION
**Pull request by Lolimkeri**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Tysyatsky — APPROVED** [review]
The fix is working.
image

**Review by MaksBrat — APPROVED** [review]

File: src/features/AdditionalPages/NewsPage/News.styles.scss [hunk 1]
```diff
@@ -24,38 +24,42 @@
...
+            display: initial;
+            width: 100%;
+        }
+    
+        .newsImage {
+            margin-bottom: f.pxToRem(20px);
+            margin-left: f.pxToRem(20px);
+            height: 100%;
+            max-height: fit-content;
+            min-width:fit-content;
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/News.styles.scss:56]: space

**Review by Katerix — COMMENTED** [review]
generally fine

File: src/features/AdditionalPages/NewsPage/News.styles.scss (hunk 1 above)
Comment (Lolimkeri) [reply src/features/AdditionalPages/NewsPage/News.styles.scss]: done

**Review by DanyilTerentiev — APPROVED** [review]
Approved

**Pull request merged**